config.go

```diff
@@ -901,6 +905,14 @@ func loadConfig() (*config, []string, error) {
 		return nil, nil, err
 	}
 
+	// !--nocfilters and --dropcfindex do not mix.
```

Comment:
Not sure why the previous one has an exclamation point.  None of the others do.  Please remove it and the other one above too.

Reply:
It's because this is an index enabled by default.  The previous check for the addrexistsindex also uses the !, because keeping the index enabled while also dropping the index doesn't make sense.